libs/sdk-ui-kit/styles/scss/SVGicons.scss

```diff
@@ -41,6 +41,22 @@ $base_path: "~@gooddata/sdk-ui-kit/styles";
     }
 }
 
```

Comment:
Please update the copyright of this file as well. As we are making some changes now, the copyright should change accordingly, shouldn't it?

`// (C) 2007-2020 GoodData Corporation`

↓

`// (C) 2007-2022 GoodData Corporation`

Reply:
fixed